packages/ui-extensions-react/src/surfaces/customer-account/hooks/settings.ts

```diff
@@ -8,8 +11,9 @@ import {useSubscription} from './subscription';
  */
 export function useSettings<
   Settings extends ExtensionSettings,
+  Target extends RenderOrderStatusExtensionTarget = RenderOrderStatusExtensionTarget,
```

Comment:
I don't know where we have the list of apis that will come to customer account extensibility, but do you know if settings will become part of standard api eventually ?